.travis.yml

```diff
@@ -25,8 +35,4 @@ notifications:
     on_success: change  # options: [always|never|change] default: always
     on_failure: always  # options: [always|never|change] default: always
     on_start: never     # options: [always|never|change] default: always
-cache:
-  apt: true
-  directories:
-  - node_modules/
-  - public/lib/
+    
```

Comment:
@ilanbiala I removed it in the beginning when working on this PR for node v4 compatibility
I re-added it again now.